### PR TITLE
Seed data into Firebase

### PR DESCRIPTION
**Pull request by catkins**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "start": "ember server",
     "build": "ember build",
-    "test": "ember test"
+    "test": "ember test",
+    "seed": "node seed.js"
```
Comment (catkins) [inline package.json:14]: I'm sure that there is a better way of doing this, but I don't know how to mount stuff like that into ember.. Perhaps there is some kind of addon?

**Comment by catkins** [comment]
@poteto I'm just going to merge this in so that I can work on the next thing before tomorrow if that's cool?

File: package.json (hunk 1 above)
Comment (poteto) [inline package.json:14]: submit a PR to ember-cli for `ember task`!

File: seed.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  },
+  {
+    name: 'New York',
+    defaultCity: 'New York City',
+    geocoderQuery: 'Times Square, New York, USA',
+    abbreviation: 'NY'
+  }
+];
+
+// fail fast!
```
Comment (poteto) [inline seed.js:28]: nice touch!

**Pull request merged**